arkane/mainTest.py

```diff
@@ -55,6 +56,11 @@ def setUpClass(cls):
         cls.base_path = os.path.join(os.path.dirname(os.path.dirname(rmgpy.__file__)), 'examples', 'arkane')
         cls.failed = []
         cls.example_types = ['species', 'reactions', 'explorer', 'networks']
+        ch2ooh_path = os.path.join(cls.base_path,'species','CH2CHOOH','CH2CHOOHscans')
+        if not os.path.exists(ch2ooh_path):
+            ch2ooh_path = os.path.join(cls.base_path,'species','CH2CHOOH','CH2CHOOHscans.zip')
+            zip_ref = zipfile.ZipFile(ch2ooh_path,'r')
+            zip_ref.extractall()
```

Comment:
Great!
Do we also delete these files after tests?

Reply:
I've adjusted it so it will delete the scans directory